react-validated-input/react-validated-input/src/ValidatedInput.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+
+export function ValidatedInput() {
+  const [password, setPassword] = useState('');
+  const [confirmedPassword, setConfirmedPassword] = useState('');
+
+  const checkMark = <i>&#9989;</i>;
+  const xMark = <i>&#10060;</i>;
+
+  const capitalLetterChecker = /([A-Z])/g;
+  const digitChecker = /\d/g;
+  const specialCharacterChecker = /[\\!\\@\\#\\$\\%\\^\\&\\*\\(\\)]/g;
+
+  let isValid = false;
+  let message = '';
+
+  if (password.length === 0) {
+    message += ' You must enter a password.';
```

Comment:
It's odd that you need a space in front of these. Hmm, I don't see that you need the space. Is it for styling? Can you do it with CSS instead?

Reply:
There was a point in time where instead of showing one message I showed all of them together then when I changed it I forgot to change the strings